apps.yaml

```diff
@@ -0,0 +1,117 @@
+aiidalab-lsmo:
+  categories:
+  - classical
+  - quantum
+  git_url: https://github.com/lsmo-epfl/aiidalab-epfl-lsmo.git
+  metadata:
```

Comment:
Yes, we need to use explicit declaration until the metadata files are updated in the app repositories. The metadata schema is largely the same, but I removed the `version` key, because it does not make much sense. We should also wait with these updates until #60 is finalized.

Reply:
I just realized that in this PR, I have actually not changed the metadata definition yet. So I am going use `$ref` instead.